tools/lxdclient/client.go

```diff
@@ -251,6 +242,46 @@ func newRawClient(remote Remote) (*lxd.Client, error) {
 	return client, nil
 }
 
+// Generate host string from remote.ID and remote.Host
+func generateHostString(id string, host string) (string) {
+	if host != "" {
+		is_localhost, _ := isLocalhost(host)
```

Comment:
I don't think there's a compelling reason to swallow the error. If InterfaceAddrs returns an error, something's probably very wrong. Can we please just bubble it up?